libs/block-scripts/scripts/codegen.js

```diff
@@ -2,12 +2,17 @@ import { writeFileSync } from "node:fs";
 import path, { join } from "node:path";
 
 import { generateTypeScriptFromEntityType } from "@blockprotocol/graph/codegen";
-import { validateVersionedUrl } from "@blockprotocol/type-system/slim";
 import chalk from "chalk";
 import fs from "fs-extra";
 
 import { blockRootDirPath } from "../shared/paths.js";
 
+/**
+ * @param {string} url
+ * @returns {boolean}
+ */
+const isVersionedUrlValid = (url) => /(.+\/)v\/(\d+)(.*)/.test(url);
```

Comment:
The `type-system` package has DOM typings in it. This package only has `Node` types. Technically it is correct for this to be an error because `type-system` might use APIs that are unavailable in this command line tool, and there is no building or polyfilling going on here. So it would be inaccurate to just add the DOM types to the TS config for this package.

I've opted to just do a minimal URL validation instead of sinking more time into worrying about it.